docs/list.yaml

```diff
@@ -50,6 +50,42 @@ PHP:
       description: Adds CORS headers support in your Laravel application
       url: https://github.com/barryvdh/laravel-cors
       tags: ['Laravel', 'CORS']
+    barryvdh/laravel-debugbar:
+      description: Laravel Debugbar (Integrates PHP Debug Bar)
+      url: https://github.com/barryvdh/laravel-debugbar
+      tags: ['Laravel', 'Debugging']
+    jenssegers/mongodb:
+      description: A MongoDB based Eloquent model and Query builder for Laravel (Moloquent)
+      url: https://github.com/jenssegers/laravel-mongodb
+      tags: ['Database', 'Moloquent']
+    robmorgan/phinx:
+      description: Simple PHP Database Migrations
+      url: https://github.com/cakephp/phinx
+      tags: ['Migrations']
+    phpoffice/phpspreadsheet:
+      description: A pure PHP library for reading and writing spreadsheet files
+      url: https://github.com/jenssegers/laravel-mongodb
+      tags: ['Office']
+    spatie/laravel-activitylog:
+      description: Log activity inside your Laravel app https://docs.spatie.be/laravel-activitylog
+      url: https://github.com/spatie/laravel-activitylog
+      tags: ['Laravel', 'Activity Log', 'Logging']
+    vinkla/laravel-hashids:
+      description: A Hashids bridge for Laravel (generate short unique ids from integers)
+      url: https://github.com/vinkla/laravel-hashids
+      tags: ['Laravel', 'Hashids']
```

Comment:
```suggestion
      tags: ['Laravel']
```

Reply:
```Generators``` is too general, doesn't specify any certain practical appliance realm. When I apply same tag to a package, that converts numbers to string and back, and to some tool, that generates code from swagger or ERD - they, actually have almost nothing common. When I select this tag - I even don't imagine, what kind of functionality it will be related with, thus it will be useless for search - because it doesn't provide enough relevance for my search - packages, that will be filtered have very week relevance with each other.

Compare it with ```Code Generators``` - you can easily imagine what you can expect in results.